### PR TITLE
Added enrolled-pi eventhub connection uri to idpay kv

### PR DESCRIPTION
**Pull request by petretiandrea**
- This PR allow to create a connection uri string to idpay key vault in order to connect with enrolled-payment-instrument event hub

### List of changes

- added data source to get primary connection string from event hub
- added keyvault resource which format the connection string into a JAAS connection string

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/domains/idpay-app/01_keyvault.tf [hunk 1]
```diff
@@ -15,3 +25,12 @@ resource "azurerm_key_vault_secret" "aks_apiserver_url" {
 
   key_vault_id = data.azurerm_key_vault.kv.id
 }
+
+resource "azurerm_key_vault_secret" "enrolled_pi_producer_connection_uri" {
+  key_vault_id = data.azurerm_key_vault.kv.id
+  name         = "${var.domain}-enrolled-pi-producer-connection-uri"
+  value = format(
```
Comment (diegolagospagopa) [inline src/domains/idpay-app/01_keyvault.tf:32]: Please use the interpolation syntax

File: src/domains/idpay-app/env/dev/terraform.tfvars [hunk 2]
```diff
@@ -52,3 +52,9 @@ enable = {
     eventhub = true
   }
 }
+
+#
+eventhub_enrolled_pi = {
+  namespace_name      = "cstar-d-evh-ns"
+  resource_group_name = "cstar-d-msg-rg"
+}
```
Comment (diegolagospagopa) [inline src/domains/idpay-app/env/dev/terraform.tfvars:60]: Please enable in your editor the feature that allow to put an empty row in each file

Comment (diegolagospagopa) [inline src/domains/idpay-app/env/dev/terraform.tfvars:60]: Please add the same value for UAT and PROD, to avoid problems during the apply

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: src/domains/idpay-app/01_keyvault.tf (hunk 1 above)
Comment (petretiandrea) [reply src/domains/idpay-app/01_keyvault.tf:32]: Instead of format?

Comment (diegolagospagopa) [reply src/domains/idpay-app/01_keyvault.tf:32]: Yes please

Comment (petretiandrea) [reply src/domains/idpay-app/01_keyvault.tf]: Done

**Comment by GiovanniMancini** [comment]
@petretiandrea Could you please resolve conflicts?

**Pull request merged**